backend/src/main.ts

```diff
@@ -5,7 +5,12 @@ import { createServer } from "http";
 import { Server } from "socket.io";
 
 const httpServer = createServer();
-const io = new Server(httpServer);
+const io = new Server(httpServer, {
+    cors: {
+        origin: "http://localhost:3000",
```

Comment:
Just an idea: but we could switch to https for local dev as well. I was using https://www.npmjs.com/package/vite-plugin-mkcert lately to prevent some dev problems

Reply:
Nice idea, I've created #12 so that we don't forget about it.